Ghidra/Features/Python/ghidra_scripts/ImportFunctionsAndLabels.py

```diff
@@ -0,0 +1,35 @@
+# Imports a file with lines in the form "symbolName 0xADDRESS function_or_label" where "f" indicates a function and "l" a label
+# @author matedealer <git@matedealer.de>
+# @category Data
+#
+
+from ghidra.program.model.symbol.SourceType import *
```

Comment:
You should add
from __future__ import print_function
and use only print functions to be compatible with Python3. Python2 is "end of life" soon.

Reply:
You don't have to compare something to none in python. None itself is falsy, you can just use if func: instead of if func != None